app/src/main/kotlin/com/hippo/ehviewer/ui/main/DownloadCard.kt

```diff
@@ -119,12 +123,13 @@ fun DownloadCard(
                                 overflow = TextOverflow.Ellipsis,
                             )
                         }
+                        val ratingBasePadding = (LocalTextStyle.current.lineHeight.value.dp - dimensionResource(id = R.dimen.rating_size)) / 2
```

Comment:
`TextStyle.lineHeight`'s value is `sp`, we shouldn't use it as `dp` directly.
Find a better approach.

Reply:
```kotlin
with(LocalDensity.current) {
    LocalTextStyle.current.fontSize.toDp()
}
```
Will use something like this instead.